Lombiq.Tests.UI/Constants/Paths.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+
+namespace Lombiq.Tests.UI.Constants;
+
+public static class Paths
+{
+    public const string DefaultSetupSnapshotDirectoryPath = "SetupSnapshot";
+    public const string TempDirectoryPath = "Temp";
+    public const string ScreenshotsDirectoryName = "Screenshots";
+
+    public static string GetTempSubDirectoryPath(string contextId, string subDirectoryName) =>
```

Comment:
I see you pass `string.Empty` to this [below](https://github.com/Lombiq/UI-Testing-Toolbox/pull/152/files#diff-7296bb52a59df5fa30fd8222aa55a27fb459709e7e0cb6539be92a6ac2606069R159). Would it be nicer to make `""` the default value of the `subDirectoryName` argument? Or better yet, make it `params string[] subDirectoryNames`?